sycl/test/inline-asm/negative_tests/asm_unsup_instruction_divm_skl.cpp

```diff
@@ -0,0 +1,33 @@
+// UNSUPPORTED: cuda
+// REQUIRES: gpu,linux
+// RUN: %clangxx -fsycl %s -DINLINE_ASM -o %t.out
+// RUN: not %t.out 2>&1 | FileCheck %s
+
+// CHECK: Build succeeded
```

Comment:
I wonder why do we have tests that expected to be successfully built in `negative_tests` directory?

If these tests require particular HW to catch device program build error, than I suggest to remove them as they are too fragile, IMHO